tox.ini

```diff
@@ -82,5 +82,5 @@ deps=
   teamcity-messages
 changedir=packages/bootstrap/extra
 commands=
-  python setup.py install
+  pip install .
```

Comment:
To maintain consistency with `packages/bootstrap/build` using `pip3` might be a good idea.